packages/backend/src/plugins/azure-sites.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createRouter,
+  AzureSitesApi,
+} from '@backstage/plugin-azure-sites-backend';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  return await createRouter({
+    logger: env.logger,
+    azureSitesApi: AzureSitesApi.fromConfig(env.config),
+    permissions: env.permissions,
```

Comment:
a hint: if you declare this instead (especially in installation instructions) as:

```ts
return await createRouter({
  ...env,
  <other specific things, e.g. the azureSitesApi>
});
```

then people will be better prepared for the addition of new dependencies in the future without breakages, since you spread the entire env by default.